misk/src/main/kotlin/misk/web/WebConfig.kt

```diff
@@ -30,7 +30,10 @@ data class WebConfig(
   val queue_size: Int? = null,
 
   /** Maximum number of threads in Jetty's thread pool. */
-  val jetty_max_thread_pool_size: Int? = null,
+  val jetty_max_thread_pool_size: Int = 200,
```

Comment:
This was the default value, and I shouldn't have made it nullable in the first place